src/styles/button.scss

```diff
@@ -17,34 +11,26 @@ $fd-button-height: 2.25rem;
 $fd-button-clickable-height: 2.75rem;
 $fd-button-touchable-area: ($fd-button-clickable-height - $fd-button-height) * 0.5;
 $fd-button-padding-x: calc(0.625rem - #{$fd-button-border-width});
-$fd-button-min-width: 2.25rem;
-$fd-button-icon-margin: 0.375rem;
 
 $fd-button-compact-height: 1.625rem;
 $fd-button-compact-clickable-height: 2rem;
 $fd-button-compact-touchable-area: ($fd-button-compact-clickable-height - $fd-button-compact-height) * 0.5;
 $fd-button-compact-padding-x: calc(0.5rem - #{$fd-button-border-width});
-$fd-button-compact-min-width: 2rem;
-$fd-button-compact-line-height: calc(#{$fd-button-compact-height} - 2 * #{$fd-button-border-width});
 
-$fd-menu-button-max-width: 12rem;
-
-$block: #{$fd-namespace}-button;
+.#{$block-button} {
```

Comment:
I think we use just `$block` naming for the root selector

Reply:
file where this variable is defined is imported by the many other files, it's done to avoid any collisions.